pkg/test/framework/environments/kubernetes/implementation.go

```diff
@@ -113,7 +119,38 @@ func (e *Implementation) Initialize(ctx *internal.TestContext) error {
 	if err := e.systemNamespace.allocate(); err != nil {
 		return err
 	}
-	return e.dependencyNamespace.allocate()
+	if err := e.dependencyNamespace.allocate(); err != nil {
+		return err
+	}
+
+	if e.kube.DeployIstio {
+		goDir := os.Getenv("GOPATH")
+		chartsDir := path.Join(goDir, "src/istio.io/istio/install/kubernetes/helm")
+
+		// TODO: We should pass a dynamic system namespace.
+		// TODO: Values files should be parameterized.
+		namespace := "istio-system"
```

Comment:
Need to use `e.systemNamespace` here

Reply:
That doesn't work at this point. I am separately investigating: https://github.com/istio/istio/pull/8828
I'll fix it separately after the investigation.